# message for passing robot command button presses throughout the gui
string command
uint8 robot_id